functions/callback-functions.md

```diff
@@ -0,0 +1,214 @@
+# Callback Functions
+
+## Goal
+
+One of the ways we'll solve problems in JavaScript is using callback functions. Getting the hang of callback functions takes practice.
+
+The goal of this lesson is to introduce callback functions, so we can use this vocabulary as we see functions passed around as arguments in the future.
+
+## Another Way to Look at Functions
+
+We might have been in situations before where we thought, "Ugh! This logic is reusable. The only thing that needs to change is one piece of behavior." If we've ever thought, "If only we could take in behavior as an argument," we're in luck!
+
+For example, let's imagine this problem and pseudocode:
+
+We need to build one calculator function that can add, subtract, multiply, and divide two numbers.
+
+In order to accomplish that goal, we need three arguments:
+
+1. The first number
+2. The second number
+3. Something that represents to operation we need, to help us determine if we're adding, subtracting, multiplying, or dividing
+
+We could approach the third argument by expecting a string, such as `'add'`, `'+'`, `'multiply'`, or `'*'`.
+
+<!-- prettier-ignore-start -->
+```js
+const calculator = function(a, b, operation){
+  // if (operation === 'add') {
+  //  return a + b;
+  // }
+}
+```
+<!-- prettier-ignore-end -->
+
+However, wouldn't it be grand if we could _pass in_ the actual operation, whether it's addition or subtraction, and execute it?
+
+## Callback Functions
+
+In order to pass around behavior, we pass around functions.
+
+More specifically, a function can _take in a function_ as an _argument_, and then call that function.
+
+To help us talk about functions, we can use specific vocabulary around these functions. A **callback function** is a nickname for a function that is _passed into_ another function.
+
+We can reverse that definition and imagine a function that _takes in another function as an argument_. The function passed in as an argument can be labeled as a _callback function_.
```

Comment:
Consider paring this back. Having the bouncing back and forth between perspectives feels simultaneously redundant and confusing to me.
```suggestion
A **callback function** is a term for a function that is _passed into_ another function.
```